### PR TITLE
Bump tf version in examples

### PR DESCRIPTION
**Pull request by seanpmorgan**
Updating tf dependency versions

### CODE REVIEW

**Comment by seanpmorgan** [comment]
Not sure how to make this diff not the full notebook but the only thing modified should be the tf version...

**Comment by facaiy** [comment]
@yifeif Hi, Yifei, if I remember it correctly, you said that tf-team is making GPU pip package the default `tensorflow` package in releases after 1.14. Do it work for tf 2.0 as well? If yes, we can use `pip install tensorflow==2.0.0-beta1` for GPU examples, right?

**Comment by seanpmorgan** [comment]
Quick check looks like no.. but will wait for a confirmation. I would have thought you'd be correct but maybe it's not part of tf2 yet.

```
sean-morgan@ubuntu-spmorgan:~$ pip install tensorflow==2.0.0-beta1
sean-morgan@ubuntu-spmorgan:~$ python
Python 3.6.8 |Anaconda, Inc.| (default, Dec 30 2018, 01:22:34) 
>>> import tensorflow as tf
>>> tf.test.is_gpu_available()
False
>>> tf.test.is_built_with_cuda()
False

```

**Comment by facaiy** [comment]
> I would have thought you'd be correct but maybe it's not part of tf2 yet.

Thanks for your verification, Sean. Maybe it's still in process

File: examples/layers_weightnormalization.ipynb [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "nbformat_minor": 0,
   "metadata": {
     "colab": {
-      "name": "layers_weightnormalization.ipynb",
+      "name": "Copy of layers_weightnormalization.ipynb",
```
Comment (WindQAQ) [inline examples/layers_weightnormalization.ipynb]: Sean, does this name matter?

File: examples/optimizers_lazyadam.ipynb [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "nbformat_minor": 0,
   "metadata": {
     "colab": {
-      "name": "optimizers_lazyadam.ipynb",
+      "name": "Copy of optimizers_lazyadam.ipynb",
```
Comment (WindQAQ) [inline examples/optimizers_lazyadam.ipynb]: ditto

File: examples/layers_weightnormalization.ipynb (hunk 1 above)
Comment (seanpmorgan) [reply examples/layers_weightnormalization.ipynb]: Hmmm honestly don't know, but I'll revert it just for cleanliness

File: examples/optimizers_lazyadam.ipynb (hunk 2 above)
Comment (seanpmorgan) [reply examples/optimizers_lazyadam.ipynb]: Done. Thanks!

**Comment by facaiy** [comment]
@WindQAQ Tzu-Wei, feel free to merge the PR if all your comments are solved :-)

**Pull request merged**